gbdxtools/images/meta.py

```diff
@@ -483,7 +495,16 @@ def plot(self, spec="rgb", **kwargs):
                 cmap = "Greys_r"
             self._plot(tfm=self._single_band, cmap=cmap, **kwargs)
         else:
-            self._plot(tfm=getattr(self, spec), **kwargs)
+            if spec == "rgb" and self._has_token(**kwargs):
+                self._plot(tfm=self.base_layer_match, **kwargs)
+            else:
+                self._plot(tfm=getattr(self, spec), **kwargs)
+
+    def _has_token(self, **kwargs):
+        if "access_token" in kwargs or "MAPBOX_API_KEY" in os.environ:
```

Comment:
Are MAPBOX_API_KEYs and DigitalGlobe maps api keys the same?

Reply:
well good point, yes and no. A maps api is in fact just a mapbox api key by nature. And in our prod env our maps api token is set as MAPBOX_API_KEY env var. You could potentially see errors where a user has a mapbox env var set and it doesnt have permissions to access maps api.